js/tests/unit/offcanvas.spec.js

```diff
@@ -287,6 +361,38 @@ describe('OffCanvas', () => {
       }
     })
 
+    it('should throw error on protected method', () => {
+      fixtureEl.innerHTML = '<div></div>'
+
+      const div = fixtureEl.querySelector('div')
+      const action = '_getConfig'
+
+      jQueryMock.fn.offcanvas = OffCanvas.jQueryInterface
+      jQueryMock.elements = [div]
+
+      try {
+        jQueryMock.fn.offcanvas.call(jQueryMock, action)
+      } catch (error) {
+        expect(error.message).toEqual(`No method named "${action}"`)
+      }
```

Comment:
IMO, you should use `toThrowError` to keep the consistency in the code base and to check the exception type as well.

Reply:
is easy to change it. I got it for tooltip and modal tests